deploy_connect_web_js.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
deploy_connect_web_js.sh => release_connect_web_js.sh (I have the feeling that release is a bit more correct here)